### PR TITLE
fdisk test case for disk partition and format file system

### PR DESCRIPTION
**Pull request by nareshkamboju**

### CODE REVIEW

**Comment by nareshkamboju** [comment]
Tested on LAVA with these changes
https://lkft.validation.linaro.org/scheduler/job/1610946#L1454

results link,
https://lkft.validation.linaro.org/results/1610946

File: automated/lib/sh-test-lib [hunk 1]
```diff
@@ -476,3 +476,85 @@ generate_skipfile() {
...
+    echo w) | fdisk "${device}"
+    check_return "fdisk-${device}-end-partition"
+    # sync and wait 5 sec
+    sync
+    sleep 5
+}
+# Usage:
+# format_partition "/dev/sdb" "ext4"
+# shellcheck disable=SC2039
+format_partition() {
```
Comment (mwasilew) [inline automated/lib/sh-test-lib:536]: shouldn't it be named 'format_partitions' ? The code suggests it formats all partitions in the device.

File: automated/linux/fdisk/fdisk.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                    [-z <+10G +5G>]" 1>&2
+    exit 1
+}
+
+# create_partitions "/dev/sdb" "+5G +5G"
+create_partitions() {
+    echo
+    echo "Creating partition: ${DEVICE}"
+    skip_list="format-partitions"
+    partition_disk "${DEVICE}" "${SIZES}"
```
Comment (mwasilew) [inline automated/linux/fdisk/fdisk.sh:28]: please check if SIZES is defined and not empty.

Comment (nareshkamboju) [reply automated/linux/fdisk/fdisk.sh:28]: if SIZES="" then single partition with total available size.

File: automated/lib/sh-test-lib (hunk 1 above)
Comment (nareshkamboju) [reply automated/lib/sh-test-lib:536]: right.

**Comment by nareshkamboju** [comment]
@mwasilew Thanks for the review.
I have addressed your review comments and uploaded in version of patch for pull request.

**Pull request merged**